lib/takeaway.rb

```diff
@@ -0,0 +1,40 @@
+class Takeaway 
+  attr_reader :basket, :menu, :grand_total
```

Comment:
do you need the attr_reader for menu given that read_menu already returns @menu?